src/commands/mod.rs

```diff
@@ -26,19 +24,18 @@ pub use whoami::whoami;
 
 /// Run the given command and return its stdout.
 pub fn run(mut command: Command, command_name: &str) -> Result<(), failure::Error> {
-    info!("Running {:?}", command);
+    log::info!("Running {:?}", command);
 
     let status = command.status()?;
 
-    if status.success() {
-        Ok(())
-    } else {
+    if !status.success() {
         failure::bail!(
-            "failed to execute `{}`: exited with {}",
-            command_name,
-            status
+            "command exited with {}\n`{}`",
+            status,
+            command_name.replace("\"", "")
```

Comment:
Why does the `command_name` have quotes?

Reply:
I'm not entirely sure why it has quotes, I think it has something to do with the implementation of `Display` on `Command` (although I'm not sure). After this the quotes are removed though - I don't think we are the ones that implemented the quotes in the first place.